tests/test_samples.py

```diff
@@ -59,6 +59,10 @@ def test_build_mcmc():
     for p in params:
         assert(mcmc.limits[p] == limits[p])
 
+    ns = NestedSamples(data=samples, logzero=-1e25)
+    assert(len(ns) == nsamps)
+    assert_array_equal(ns.columns, numpy.array([0, 1, 2], dtype=object))
+
```

Comment:
Could you perhaps also add some tests here with a `logL_birth` and `logL` argument, with a check that the logzeros are being modified correctly?

Reply:
Added test in [1e85d65](https://github.com/williamjameshandley/anesthetic/pull/71/commits/1e85d65efa54307bd4bafc368d78b91c5302885f) where I modified `logL` such that it has some `-1e300` values, which get turned into `-numpy.inf`.

Good call. Had to move the conversion, because it was previously only affecting calls to NestedSamples with root passing.

Reply:
Excellent. Ideally we should also check that if you pass ``logzero=-1e301`` it doesn't change anything, and also that the rest of the logL column is unchanged.

Reply:
Done in [2a16556](https://github.com/williamjameshandley/anesthetic/pull/71/commits/2a1655660982d74f9682076ddff53d882841c384).